justfile

```diff
@@ -74,3 +74,13 @@ build-android:
 [working-directory: 'app']
 build-ios:
 	flutter build ios --no-codesign
+
+# analyze Dart code
+[working-directory: 'app']
+analyze-dart:
+    flutter analyze
```

Comment:
Most likely, this needs to be prefixed with `fvm`, right?

Reply:
This is a bit of an open question, I should have brought that up earlier. Right now, the CI installs the Flutter version specified in the `.fvmrc` file. However, it doesn't install `fvm` as such, and therefore we don't need to prefix the command with `fvm` for it to work.
While that's fine for the CI, we as developers need to make a choice: Either we make sure we have the right flutter version installed, or we use `fvm`. If we use `fvm`, we should use an alias, otherwise we can't use the `justfile`. The recommendation is to have an alias like `flutter = flutter fvm`.

The alternative to the above would be to install & use `fvm` on the CI. The downside is that every developer has to use `fvm` in that case.

Reply:
Thank you for the explanation. Let's keep it as is then.